rotkehlchen/tests/utils/dataimport.py

```diff
@@ -2370,8 +2370,59 @@ def assert_bittrex_import_results(rotki: Rotkehlchen):
         fee_currency=A_BTC,
         link='Imported bittrex trade 28445361-31c6-1ab1-91a9-e196d50ad1a5 from csv',
         notes='Bought 731.28354007 SWT for 0.24999841 BTC',
+    ), Trade(
+        timestamp=Timestamp(1546269766),
+        location=Location.BITTREX,
+        base_asset=A_BTC,
+        quote_asset=A_USDT,
+        trade_type=TradeType.BUY,
+        amount=AssetAmount(FVal('0.01')),
+        rate=Price(FVal('10')),
+        link='Imported bittrex trade aaaaaa from csv',
+    ), Trade(
+        timestamp=Timestamp(1546269967),
+        location=Location.BITTREX,
+        base_asset=A_ETH,
+        quote_asset=A_USDT,
+        trade_type=TradeType.BUY,
+        amount=AssetAmount(FVal('1')),
+        rate=Price(FVal('1')),
```

Comment:
Here and where you use `FVal(1)` use the constant `ONE` that we already have defined